src/iuse.cpp

```diff
@@ -1934,7 +1934,7 @@ std::optional<int> iuse::extinguisher( Character *p, item *it, const tripoint &
     // Also spray monsters in that tile.
     if( monster *const mon_ptr = get_creature_tracker().creature_at<monster>( dest, true ) ) {
         monster &critter = *mon_ptr;
-        critter.moves -= to_moves<int>( 2_seconds );
+        critter.mod_moves( -to_moves<int>( 1_seconds ) );
```

Comment:
2 to 1 second?

Reply:
These first three were all errors on my part, copy-pasting and not setting the old value